core/observables/file.py

```diff
@@ -34,3 +36,39 @@ def info(self):
         i['hashes'] = self.hashes
         i['filenames'] = self.filenames
         return i
+
+
+class Certificate(Observable):
+
+    value = StringField(verbose_name="Value")
+    hashes = ListField(DictField(), verbose_name="Hashes")
+    body = ReferenceField("AttachedFile")
+
+    exclude_fields = Observable.exclude_fields + ['body']
+
+    @staticmethod
+    def check_type(txt):
+        return True
+
+    @classmethod
+    def from_data(cls, data, hash_sha256):
+        body = AttachedFile.from_content(
+            BytesIO(data), "cert.der", "application/pkix-cert",
+        )
+        return cls.get_or_create(
+            value="CERT:{}".format(hash_sha256), body=body,
+        )
+
+
+class CertificateSubject(Observable):
+
+    value = StringField(verbose_name="Text subject")
+
+    @staticmethod
+    def check_type(txt):
+        return True
+
+    def info(self):
```

Comment:
I don't think you need to redefine `info` in this case. `value` is already present in the dict

Reply:
Totally.